src/RuleCollection.php

```diff
@@ -1199,7 +1199,7 @@ public static function previewImportRules()
             $tmprule = new $rule['sub_type']();
            //check entities
             if ($tmprule->isEntityAssign()) {
-                $entities_found = $entity->find(['completename' => $rule['entities_id']]);
+                $entities_found = $entity->find(['completename' => html_entity_decode($rule['entities_id'])]);
```

Comment:
Are we sure this is the only place where an unexpected HTML entities encoding has been done?

Reply:
Other check with ```completename``` are handle correctly (IMHO)